### Laelaps laelaps_control/SetRobotMode service
# 
# Set robot's operational mode.
#

industrial_msgs/RobotMode mode    # robot mode: AUTO MANUAL
---
int8 rc                           # return code
